src/robot/running/arguments/argumentparser.py

```diff
@@ -16,9 +16,11 @@
 from robot.errors import DataError
 from robot.utils import JYTHON, PY_VERSION, PY2
 from robot.variables import is_dict_var, is_list_var, is_scalar_var
+from pydoc import locate
```

Comment:
PEP8 import order
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```

Reply:
Thank for comments. I change import order